tests/builder/Data/ByteString/Builder/Tests.hs

```diff
@@ -165,6 +168,31 @@ testHandlePutBuilderChar8 =
         unless success (error msg)
         return success
 
+testWriteFile :: TestTree
+testWriteFile =
+    testProperty "writeFile" testRecipe
+  where
+    testRecipe :: Recipe -> Property
+    testRecipe recipe =
+        ioProperty $ do
+            (tempFile, tempH) <- openTempFile "." "test-builder-writeFile.tmp"
+            hClose tempH
+            let b = fst $ recipeComponents recipe
+            writeFile tempFile b
+            lbs <- L.readFile tempFile
```

Comment:
Please force evaluation of `lbs` before removing `tempFile`, as in other tests above.